### PR TITLE
Fixing dockerfile and run script

### PR DESCRIPTION
**Pull request by bathina2**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,14 @@ RUN go mod download
 
 COPY . .
 
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o kubestr .
```
Comment (vkamra) [inline Dockerfile]: Shouldn't this (and doesn't this already) happen outside the dockerfile?

Comment (bathina2) [reply Dockerfile]: `jobs:
  # Run tests.
  # See also https://docs.docker.com/docker-hub/builds/automated-testing/
  test:
    runs-on: ubuntu-latest

    steps:
      - name: Check out code into the Go module directory
        uses: actions/checkout@v2

      - name: Run tests
        run: docker build . --file Dockerfile`

Comment (bathina2) [reply Dockerfile]: Forgive the formatting but no it doesn't. The steps to deploy are to first download the repo and then run docker build.

File: Dockerfile [hunk 2]
```diff
@@ -9,10 +14,14 @@ RUN go mod download
 
 COPY . .
 
-RUN GOOS=linux GOARCH=amd64 go build -ldflags="-w -s" -o /bin/kubestr
+RUN go build -ldflags="-w -s" -o kubestr .
```
Comment (bathina2) [inline Dockerfile:17]: go get

**Pull request merged**